app/views/layouts/application.html.erb

```diff
@@ -3,9 +3,10 @@
   <head>
     <title>LokalerLnuNo</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="viewport" content="viewport-fit=cover" />
```

Comment:
Denne overskriver de to over, sånn at de ikke gjelder lenger. Burde kombineres til én, med kommaseparerte verdier:

```html
    <meta name="viewport" content="viewport-fit=cover,width=device-width,initial-scale=1" />
```

Ellers blir det ikke zoomet riktig inn på mobil.